rejoiner/pom.xml

```diff
@@ -258,6 +258,11 @@
        <version>0.5.1</version>
```

Comment:
there's a 0.6.1 version, any reason to not use that?

Reply:
I don't want to update a lib version in my PR. Only what is really necessary for this particular functionality. Hopefully the maintainers will upgrade libs soon...